app/views/admin/attachments/edit.html.erb

```diff
@@ -11,8 +11,8 @@
   </section>
 
   <% if attachment.html? %>
-    <div class="col-md-4">
+    <section class="govuk-grid-column-one-third">
```

Comment:
Maybe `aside` is a better tag (https://www.w3.org/html/wiki/Elements/aside)

```suggestion
    <aside class="govuk-grid-column-one-third">
```

Reply:
Agreed - makes sense. 👍